plugins/juju/juju.plugin.zsh

```diff
@@ -95,6 +95,18 @@ jaddr() {
   fi
 }
 
+# Destroy all controllers
+jclean() {
+  if (( ! ${+commands[jq]} )); then
+    echo "jq is required but could not be found." >&2
+    return 1
+  fi
+  
+  juju controllers --format=json | jq '.controllers | keys[]' | timeout 2m xargs -I {} juju destroy-controller --destroy-all-models --destroy-storage --force --no-wait -y {}
+  juju controllers --format=json | jq '.controllers | keys[]' | timeout 2m xargs -I {} juju kill-controller -y -t 0 {}
+  juju controllers --format=json | jq '.controllers | keys[]' | timeout 10s xargs -I {} juju unregister {}
```

Comment:
Can we maybe cache these instead of calling 3 times?

Reply:
Also this function is super destructive. Maybe adding a confirmation prompt before is a good idea.

Reply:
Done.